server/base/src/main/java/org/apache/accumulo/server/util/SystemPropUtil.java

```diff
@@ -76,13 +76,13 @@ private static String validateSystemProperty(String property, final String value
     if (!Property.isValidZooPropertyKey(property)) {
       IllegalArgumentException iae =
           new IllegalArgumentException("Zookeeper property is not mutable: " + property);
-      log.error("Encountered error setting zookeeper property", iae);
+      log.debug("Encountered error setting zookeeper property", iae);
```

Comment:
Consider removing or using trace.  The error logged by the calling method should be enough - and usually systems seem to run with logging set to debug because we  do not seem to log enough information at info or above to adequately troubleshoot issues when they occur.  (Which is a whole other topic) Sorry I was not clearer on my earlier comment.

Reply:
I agree that `debug` is commonly used for "general awareness" instead of `Info`.

So I'm happy to switch to `trace` but I'd like to also switch all instances of `debug` in SystemPropUtil if they are already logged in their calling methods. 

Outside of this PR, maybe a overhaul of logging should be considered to help reduce the need for `debug` use in production environments?

Reply:
I would be fine with switching them to trace if they are logged by the calling methods.

A review / overhauling of the logging is probably overdue, but also too large of a task to handle at one time.  One factor is there are down-stream users that may be processing the logs and it is really difficult to know exactly what information is important at a given time.  Most of the time, it seems like noise. Until that one time you need to figure out X and then you start wishing there were more details ;-)

There is a start at unifying things with the  org.apache.accumulo.core.logging.Logging class.  Using and extending that could help reduce the scope of the problem moving forward.